Trustroots/trustroots
# Docker compose update

#### Proposed Changes

S'all in the commits...

* Remove maildev container: maildev already runs in the main container with Trustroots app
* Remove nginx service
* Upgrade docker-compose syntax from ? to 3.7 (current)
* Tweak a couple of commands here and there
* Updates to docs
* Copy both `package.json` and `package-lock.json` for `node_modules` cache layer busting
* Remove redundant `imagemagick`
* Remove InfluxDB container: this might be nice to add back in follow-ups, but for now the one we used was deprecated.
* Expose more ports

#### Testing Instructions

* Ensure you have `host: "0.0.0.0"` in your `config/env/local/js` file
* If you had previous instances, you might wanna `docker-compose build` again
* `docker-compose up`
* works?
